workshop/4-auth-me-baby.md

```diff
@@ -45,25 +45,29 @@ Now we need to add the `signin` and `signout` actions to the application route.
 Let's generate the route.
 
 ```
-ember g route Application
+ember g route application
```

Comment:
I've already generated this route in the addon section, so you can remove these instructions.